.github/actions/launchable/setup/action.yml

```diff
@@ -96,12 +96,19 @@ runs:
         echo "LAUNCHABLE_TOKEN=${{ inputs.launchable-token }}" >> $GITHUB_ENV
       if: steps.enable-launchable.outputs.enable-launchable
 
+    - name: Set up path
+      shell: bash
+      working-directory: ${{ inputs.srcdir }}
+      # Since updated PATH variable will be available in only subsequent actions, we need to add the path beforehand.
+      # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path
+      run: echo "$(python -msite --user-base)/bin" >> $GITHUB_PATH
+      if: startsWith(inputs.os, 'macos')
```

Comment:
Maybe `if: steps.enable-launchable.outputs.enable-launchable && startsWith(inputs.os, 'macos')`?

Reply:
Oops, good catch. Let me create the PR.

Reply:
https://github.com/ruby/ruby/pull/11185